docs/how-to/how-to-use.md

```diff
@@ -35,7 +35,70 @@ Click anywhere to go to the login page.
 
 ## Login
 
-For now, you can log in with any credentials as the logic is not developed yet.
+### You need to have a registered user to access to yugi academy
+
+You can login to the web page with a previously registered user.
+Here, you can also go to sign up page (sign up button) to register a new user.
+
+### Information required for login
+
+- Username
+
+- Password
+
+---
+
+## Sign up
+
+### Information required for sign up
+
+- Username
+
+- Email
```

Comment:
Delete mail here, since the user will only use the basic credentials (username and password). The email goes to the user profile.